pacehrh/vignettes/pacehrh.rmd

```diff
@@ -141,15 +141,15 @@ stochastic distributions around various parameter values. See the table below fo
-| Mortality rates                                               | Uniform                  | mean, p          | min = mean - mean\*p <br /> max = mean + mean\*p |
-| Incidence rates <br />(**TaskValues** column H)               | Uniform                  | mean, p          | min = mean - mean\*p <br /> max = mean + mean\*p |
-| Annual delta fertility rates                                  | Truncated normal*        | delta, p, q      | mean = delta <br /> std dev = p <br /> a = mean - p\*q <br /> b = mean + p\*q  |
-| Annual delta mortality rates                                  | Truncated normal*        | delta, p, q      |  mean = delta <br /> std dev = p <br /> a = mean - p\*q <br /> b = mean + p\*q | 
-| Annual delta incidence rates <br />(**TaskValues** column L)  | Truncated normal*        | delta, p, q      |  mean = delta <br /> std dev = p <br /> a = mean - p\*q <br /> b = mean + p\*q |
-| Seasonality ratio to mean                                     | Uniform                  | mean, p          | min = mean - mean\*p <br /> max = mean + mean\*p |
-| Minutes per contact <br />(optional; **TaskValues** column O) | Lognormal                | mean, p          | logsd = sqrt(log((1 + p^2))) <br /> logm = (log(1 + p^2))/2 |
-| Hours per week <br />(optional; **TaskValues** column P)      | Uniform                  | mean, p          | min = mean - mean\*p <br /> max = mean + mean\*p |     
+| Fertility rates                                               | Uniform                  | mean, p          | $$min = mean - mean*p$$ $$max = mean + mean*p$$ |
+| Mortality rates                                               | Uniform                  | mean, p          | $$min = mean - mean*p$$ $$max = mean + mean*p$$ |
+| Incidence rates <br />(**TaskValues** column H)               | Uniform                  | mean, p          | $$min = mean - mean*p$$ $$max = mean + mean*p$$ |
+| Annual delta fertility rates                                  | Truncated normal*        | delta, p, q      | $$mean = delta$$ $$std.dev. = p$$ $$a = mean - p*q$$ $$b = mean + p*q$$  |
+| Annual delta mortality rates                                  | Truncated normal*        | delta, p, q      | $$mean = delta$$ $$std.dev. = p$$ $$a = mean - p*q$$ $$b = mean + p*q$$ | 
+| Annual delta incidence rates <br />(**TaskValues** column L)  | Truncated normal*        | delta, p, q      | $$mean = delta$$ $$std.dev. = p$$ $$a = mean - p*q$$ $$b = mean + p*q$$ |
+| Seasonality ratio to mean                                     | Uniform                  | mean, p          | $$min = mean - mean*p$$ $$max = mean + mean*p$$ |
+| Minutes per contact <br />(optional; **TaskValues** column O) | Lognormal                | mean, p          | $$sdlog = \sqrt{log[1 + (\frac{p}{mean})^2]}$$ $$meanlog = log(mean)-\frac{log[1 + (\frac{p}{mean})^2)]}{2}$$ |
```

Comment:
"Minutes Per Contact" looks slightly different than the original formula (p^2) is now( p/mean)^2?
also I think logarithm is usually followed by parenthesis than the square brackets?

Reply:
The formula is updated to the more general case. The one we had before is a special case when mean=1.

Reply:
And I agree that log() is more proper notation. Thank you for noticing and making the changes.